README.md

```diff
@@ -278,7 +274,7 @@ iWF allows you to start a workflow with a fixed cron schedule like below
 // or timed out, the workflow will be retried based on the retry policy. While the workflow is retrying, it won't
 // schedule its next run. If the next schedule is due while the workflow is running (or retrying), then it will skip
 that
-// schedule. Cron workflow will not stop until it is terminated or cancelled (by returning cadence.CanceledError).
+// schedule. Cron workflow will not stop until it is terminated or canceled (by returning cadence.CanceledError).
```

Comment:
cadence.CanceledError seems incorrect here given the two backends.

Reply:
Ahh right. I think we just remove this part. Returning cancel error is not supported in IWF. The only to cancel a workflow is call the stop workflow api ( the stopType is cancel by default)